app/components/personas/view.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Personas
```

Comment:
Since it's just encapsulating one reusable persona:

```suggestion
module Persona
```